schema-loader/src/main/java/com/scalar/db/schemaloader/command/StorageSpecificCommand.java

```diff
@@ -105,7 +106,12 @@ private void repairTables(
 
   @VisibleForTesting
   SchemaParser getSchemaParser(Map<String, String> options) throws SchemaLoaderException {
-    return new SchemaParser(schemaFile, options);
+    try {
+      return new SchemaParser(schemaFile, options);
+    } catch (IOException e) {
+      throw new SchemaLoaderException(
+          "Reading the schema file failed; File: " + schemaFile.toAbsolutePath(), e);
```

Comment:
```suggestion
          "Reading the schema file failed. File: " + schemaFile.toAbsolutePath(), e);
```
A bit inconsistent with other error messages in `CoreError`?

Reply:
Ah, good catch! Thanks.